src/module/actor/actions/single-check.ts

```diff
@@ -33,7 +33,7 @@ function isValidDifficultyClass(dc: unknown): dc is CheckDC | DCSlug {
     return (
         ["ac", "armor", "perception"].includes(slug) ||
         tupleHasValue(SAVE_TYPES, slug) ||
-        setHasElement(SKILL_SLUGS, slug)
+        objectHasKey(CONFIG.PF2E.skills, slug)
```

Comment:
```suggestion
        slug in CONFIG.PF2E.skills
```

Reply:
true, I forget object inclusion is not under the same restrictions as sets nor arrays, and only exists if we need to start poking in there. I changed a few others while I was at it.